pallets/runtime/tests/src/pips_test.rs

```diff
@@ -73,7 +73,9 @@ macro_rules! assert_no_pip {
 }
 
 fn make_proposal(value: u64) -> Call {
-    Call::Pips(pips::Call::set_min_proposal_deposit(value.into()))
+//    Call::Pips(pips::Call::set_min_proposal_deposit(value.into()))
+    let content = vec![b'X'; 100];
+    Call::System(frame_system::Call::remark(content)).into()
```

Comment:
Neat! I think it would be good to define a `fn dummy_call()` somewhere centrally in the testing framework we've made, so that we always use this when the call really doesn't matter.

Reply:
`remark` only works for signed origins. So we still need the other call for the root origin.

Reply:
Ah 👍